Build failed with rustc 0.13.0-dev

➜ rustc --version
rustc 0.13.0-dev (0b48001c2 2014-11-07 15:26:26 +0000)
➜ make
rustc -O -o bin/racer src/main.rs
src/racer/ast.rs:269:19: 269:28 error: attempted access of field `types` on type `&syntax::ast::PathSegment`, but no field with that name was found
src/racer/ast.rs:269         for ty in seg.types.iter() {
                                       ^~~~~~~~~
error: aborting due to previous error
Makefile:6: recipe for target 'all' failed
make: **\* [all] Error 101

--- a/src/racer/ast.rs
+++ b/src/racer/ast.rs
@@ -266,7 +266,7 @@ fn to_racer_path(pth: &ast::Path) -> racer::Path {
     for seg in pth.segments.iter() {
         let name = token::get_ident(seg.identifier).get().to_string();
         let mut types = Vec::new();    
-        for ty in seg.types.iter() {
+        for ty in seg.parameters.types().iter() {
             types.push(match ty.node {
                 ast::TyPath(ref path, _, _) => to_racer_path(path),
                 _ => panic!(format!("Cannot handle type {}", ty.node))

